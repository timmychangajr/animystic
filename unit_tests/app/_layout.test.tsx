import RootLayout from '@/app/_layout';
import { render } from '@testing-library/react-native';

jest.mock('expo-router', () => {
  return {
    ...jest.requireActual('expo-router'),
  };
});
jest.mock('@react-navigation/native', () => {
  return {
    ...jest.requireActual('@react-navigation/native'),
    useTheme: jest.fn(() => ({
      colors: {
        background: '#000'
      }
    }))
  };
});
describe('TabLayout', () => {
  test('renders correctly', async () => {
    const tree = render(<RootLayout />);
    expect(tree).toMatchSnapshot()
  });
});