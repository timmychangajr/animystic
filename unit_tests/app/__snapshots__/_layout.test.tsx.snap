// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabLayout renders correctly 1`] = `null`;
